FilteringApp/FilteringApp.Core/Models/UserValueParseResult.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace FilteringApp.Core.Models
+{
+    public class UserValueParseResult
+    {
+        //public UserValueParseResult(bool isValid, int value, string error)
```

Comment:
Please remove commented out code in case you don't plan to use it

Reply:
Removed.